Userland/Libraries/LibChess/Chess.cpp

```diff
@@ -221,42 +222,105 @@ String Move::to_algebraic() const
 
 Board::Board()
 {
-    // Fill empty spaces.
-    for (int rank = 2; rank < 6; ++rank) {
-        for (int file = 0; file < 8; ++file) {
+    load_fen("rnbqkbnr/pppppppp/8/8/8/8/PPPPPPPP/RNBQKBNR w KQkq - 0 1");
+}
+
+void Board::load_fen(String fen)
```

Comment:
Why not make this return `ErrorOr<void>`, to report invalid FEN? Using `VERIFY_NOT_REACH` and silently ignoring errors isn't very user-friendly... simply replace those with `Error::from_string_literal` with a descriptive error!

Reply:
Mostly because I'm quite new to the project and I didn't know about that. I'll definitely work on that now!